packages/react/src/components/Hero/Hero.tsx

```diff
@@ -18,7 +18,7 @@ const HeroImage: FC<ImageProps> = ({ url, alt }) => {
   const defaultSrc = sortedUrls[sortedUrls.length - 1].src;
 
   return (
-    <figure className="hero--figure">
+    <figure className="hero with-image hero--figure">
```

Comment:
"with-image" is not really BEM compliant, I think you can manage with just "hero--figure".

Reply:
@bashlk hero--figure currently exists for both if there is an image or if there isnt which is why i had to add a separate class.

Reply:
Had a quick call to discuss BEM and that we should probably refactor a bit or introduce a new modifier here.

Reply:
Made the change :)